### DISCUSSION TITLE
FR: Thino popup warning on unsaved changes

### DISCUSSION THREAD

**Discussion by lifehaschanged** [Quorafind/Obsidian-Thino#730] [Ideas 想法]
Current problem is, when writing new text to an already existing thino and then double clicking another thino, the written text of the other thino is lost because the other double clicked thino is focused.

But this could be prevented with the additions of thino alerts on unsaved changes or unsaved thinos: 

When editing thinos and someone wants to jump to other thinos with a mouse double click on a thino, a popup warning message could inform a user that the edited thino isn't saved -> popup ask for saving thino / or when thino is new and not saved -> popup ask for saving changes before editing other thinos.

 These warnings would be useful so people don't lose already written thinos.

**Comment by juestchaos** [comment]
Your issue might be due to having an older version of Obsidian installed. Please note that the installed version of Obsidian and the version that is automatically upgraded are different. You can check the detailed version number of the installed version in the "About" section.

**Comment by lifehaschanged** [comment]
I have the the newest Obsidian version 1.7.7 installed. And for testing reinstalled with a fresh 1.7.7 installer file from the obisidan.md website.
Please reproduce:
1. Open a sandbox with the sandbox command inside Obsidian
2. Install thino only
3. Create 2 thinos
4. Double click on thino 1 and write some new text in it, don't exit or save this thino
5. Double click on thino 2
6. Double click on thino 1
7. Result: The new text that was added in step 4 is not there anymore + no popup warning was there that there is some unsaved text in this thino before switching to the other saved thino

@juestchaos
This is a feature request for a popup warning on unsaved changes, not a question or bug report.
I changed the title to "FR: Thino popup warning on unsaved changes"
